### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by BQuinteros**
Added a basic layout to show the information about the weather of a city.
image

### CODE REVIEW

File: WheaterApp/app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -19,4 +19,16 @@
         <item name="android:textStyle">bold</item>
     </style>
 
+    <style name="CityAndCountryStyle">
+        <item name="android:layout_marginTop">124dp</item>
```
Comment (SartiMau) [inline WheaterApp/app/src/main/res/values/styles.xml]: no hardcode, use dimens

**Review by SartiMau — CHANGES_REQUESTED** [review]

**Review by SartiMau — DISMISSED** [review]

File: WheaterApp/app/src/main/res/layout/activity_details_city.xml [hunk 2]
```diff
@@ -1,6 +1,74 @@
...
+        app:layout_constraintTop_toBottomOf="@+id/text_view_details_activity_temp" />
+
+    <TextView
+        android:id="@+id/text_view_details_activity_description"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_top_details_activity_description"
+        android:text="@string/text_details_activity_description"
+        android:textSize="@dimen/text_size_details_activity_description"
+        android:textAlignment="center"
```
Comment (mauroLonghi) [inline WheaterApp/app/src/main/res/layout/activity_details_city.xml]: if you use in all the label's a style, in this case why did not use it?

Comment (BQuinteros) [reply WheaterApp/app/src/main/res/layout/activity_details_city.xml]: done!

**Comment by BQuinteros** [comment]
image

**Review by nicolbidone — APPROVED** [review]

**Review by lucianoeli — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Pull request merged**